components/builder-depot/src/handlers/integrations.rs

```diff
@@ -79,6 +79,10 @@ pub fn fetch_origin_integration_names(req: &mut Request) -> IronResult<Response>
         Ok(p) => p,
         Err(st) => return Ok(Response::with(st)),
     };
+    let session = req.extensions.get::<Authenticated>().unwrap().clone();
```

Comment:
we already check origin access in the validate_params helper

Reply:
Ah of course! Updated the PR :)